sherpa/utils/__init__.py

```diff
@@ -40,17 +40,22 @@
 
 # Note: _utils.gsl_fcmp and _utils.ndtri are not exported from
 #       this module; is this intentional?
-from sherpa.utils._utils import hist1d, hist2d  # type: ignore
-from sherpa.utils import _utils  # type: ignore
-from sherpa.utils import _psf    # type: ignore
-from sherpa.utils.err import IOErr
-from sherpa.utils.random import poisson_noise
-
-# We re-export the symbols from parallel but this will be removed at
-# some point.
+from ._utils import hist1d, hist2d  # type: ignore
+from . import _utils  # type: ignore
+from . import _psf    # type: ignore
+
+from .err import IOErr
+
+# We re-export symbols from sherpa.utils modules but this will be
+# removed at some point.
```

Comment:
Well,. in #2015 we saw the problem with changing symbols. This is slightly less serious here, as the symbols still exist, but to a user who accesses them it would still be a breaking change (for no obvious benefit to them). I don't know if you can use `DeprecationWarning` to say "hey, you need to access this symbol from foo.bar.baz rather than foo.bar".